docs/querying/sql.md

```diff
@@ -219,6 +219,18 @@ at execution time. To use dynamic parameters, replace any literal in the query w
 corresponding parameter value when you execute the query. Parameters are bound to the placeholders in the order in
 which they are passed. Parameters are supported in both the [HTTP POST](#http-post) and [JDBC](#jdbc) APIs.
 
+Please note that in certain cases, dynamic parameters used in expressions can cause type inference issues which can cause the query to fail. The following query is one such case:
```

Comment:
```suggestion
In certain cases, using dynamic parameters in expressions can cause type inference issues which cause your query to fail, for example:
```
Avoid "please note". Also does the type interface issue result in a specific error? If so might be nice to include it.